src/modules/authentication/AuthenticationWrapper.ts

```diff
@@ -0,0 +1,193 @@
+import {
+  AccountInfo,
+  AuthenticationResult, InteractionRequiredAuthError,
+  PopupRequest, SilentRequest
+} from '@azure/msal-browser';
+
+import { AUTH_URL, DEFAULT_USER_SCOPES, HOME_ACCOUNT_KEY } from '../../app/services/graph-constants';
+import { geLocale } from '../../appLocale';
+import { getCurrentUri } from './authUtils';
+import IAuthenticationWrapper from './IAuthenticationWrapper';
+import { msalApplication } from './msal-app';
+
+const defaultScopes = DEFAULT_USER_SCOPES.split(' ');
+const homeAccountKey = HOME_ACCOUNT_KEY;
```

Comment:
Why not use the constant directly?

Reply:
I had placed the value there and replaced it with the constant, Nice catch! Changing it